### PR TITLE
Correct grammar mistake in the file firewall docs

### PR DESCRIPTION
**Pull request by settermjd**
This relates to #2319.

### CODE REVIEW

File: modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc [hunk 1]
```diff
@@ -83,7 +83,7 @@ The full list of conditions is (File mimetype upload):
 * ends with
 * doesn’t end with
 
-NOTE: {supported-mimetype-list-url}[The complete list of available mimetypes] that ownCloud supports is available the source of ownCloud core.
+NOTE: {supported-mimetype-list-url}[The complete list of available mimetypes] which ownCloud supports is available in the ownCloud core source.
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc]: As commented in #2319, imho it looks better to have the link on onwcloud/core instead at the beginning. But if you think that my pov is ugly we can keep your PR as it is

Comment (settermjd) [reply modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc]: I think that you may have a good point. I'll play with it for 5 mins.

File: modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc [hunk 2]
```diff
@@ -83,7 +83,7 @@ The full list of conditions is (File mimetype upload):
 * ends with
 * doesn’t end with
 
-NOTE: {supported-mimetype-list-url}[The complete list of available mimetypes] that ownCloud supports is available the source of ownCloud core.
+NOTE: The complete list of available mimetypes which ownCloud supports is available {supported-mimetype-list-url}[in the ownCloud core source].
```
Comment (mmattel) [inline modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc]: Just to improve my English :smile:
Is it `in the ownCloud core source` or `in the source code of ownCloud core` ?

Comment (settermjd) [reply modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc]: They're, effectively, the same. The first is active tense, the second is passive.

**Pull request merged**